libnavigation-core/gradle.properties

```diff
@@ -1,3 +1,4 @@
 POM_ARTIFACT_ID=core
+POM_VERSION_NAME=1.0.0-SNAPSHOT
```

Comment:
We should configure it so that we only need to change `libnavigation-core` and libnavigation-ui` `gradle.properties` files as other dependent modules need to have the same version name. Does that make sense?